lib/treeherder.py

```diff
@@ -6,14 +6,14 @@
 from thclient import TreeherderClient
 
 
-def get_revision_hash(host, project, revision):
+def get_revision_hash(server_url, project, revision):
```

Comment:
This is used for Taskcluster access later. It doesn't seem to be critical and we can remove that later once we use TC again.